examples/instances_reserved_public_ip/main.tf

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2018, 2021 Oracle and/or its affiliates.
+
+terraform {
+  required_version = ">= 0.13" // terraform version below 0.12 is not tested/supported with this module
```

Comment:
As above. Why not bump this to Terraform 1.0.0?

Reply:
as above, already discussed in [previous PR](https://github.com/oracle-terraform-modules/terraform-oci-compute-instance/pull/68#discussion_r696290970)